src/shared/components/palette/complex/read-complex-field.component.ts

```diff
@@ -11,6 +11,7 @@ export class ReadComplexFieldComponent extends AbstractFieldReadComponent implem
   public paletteContext = PaletteContext;
 
   ngOnInit(): void {
+    super.ngOnInit();
```

Comment:
Do we need this for other field types too? I can only see this for 3 types now.

Reply:
Not all read components have ngOnit, whichever I have left were not being used in edit context.

Reply:
Hmm not sure I understand the _whichever I have left were not being used in edit context._ Maybe I 
 am misunderstanding the requirements

Reply:
Components like ReadEmailFieldComponent didn't implement OnInit. Since it is now inheriting AbstractFieldReadComponent which implements OnInit, I didn't explicitly add
`ngOnInit(): void {
    super.ngOnInit();
}`
to such components. Unless it is good practice to explicitly implement OnInit for components (rather than inheriting), I won't add it. Please let me know